lalrpop/src/normalize/lower/mod.rs

```diff
@@ -54,6 +54,10 @@ impl<'s> LowerState<'s> {
                     uses.push(data);
                 }
 
+                pt::GrammarItem::MatchToken(_) => {
+                    // FIXME: Do something?
```

Comment:
I think you can remove this FIXME. I don't think there's anything to do here. 

I'd replace it with a comment like this:

```
// The declarations in the match token are handled
// fully by the `token_check` when it constructs the
//  `InternToken` -- there is nothing left to do here.
```